Lib/test/_test_multiprocessing.py

```diff
@@ -258,6 +258,9 @@ def __call__(self, *args, **kwds):
 class BaseTestCase(object):
 
     ALLOWED_TYPES = ('processes', 'manager', 'threads')
+    # If not empty, limit which start method suits run this class.
+    START_METHODS: set[str] = set()
```

Comment:
Why use type annotation?
```suggestion
    START_METHODS = set()
```
or
```suggestion
    START_METHODS = ()
```
or even
```suggestion
    START_METHODS = None
```

Reply:
This form of simple type annotation is intended solely to communicate to other maintainers working on the code (and potentially their IDEs/editor) what to expect.

Reply:
This is the only annotation in this file.

Reply:
I know. and it's also a test template that commits unspeakable coding horrors in how these classes get merged and mixed in with an otherwise undeclared unittest.TestCase base class within the actual test suite runners. The value in this case is entirely for us humans given the bazillion other issues that pyright flags as it isn't able to understand _test_multiprocessing.py.